### PR TITLE
Feat: Added remove icon

### LINKED ISSUE #10: Issues: Remove feature
Added remove button in likePage and watchLater page. User can remove a particular video by clicking on the delete icon.

### PR DESCRIPTION
**Pull request by RahulSingh9131**
resolved issues related to remove icon from likePage and watchlaterPage.

### CODE REVIEW

File: src/pages/Likepage.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React,{useEffect} from 'react'
 import Footer from '../components/Footer';
 import Header from '../components/Header'
+import LikeCard from '../components/LikeCard';
 import Sidebar from '../components/Sidebar';
```
Comment (rahulcs754) [inline src/pages/Likepage.js:5]: you can create an index file and here you just write one line code 
I will explain if you're are not understand then I will explain

**Review by rahulcs754 — APPROVED** [review]
The code looks clean and good just  small suggestion

**Review by rohit8686 — APPROVED** [review]
- Functionality is working as expected
- Good to merge

File: src/pages/Likepage.js (hunk 1 above)
Comment (RahulSingh9131) [reply src/pages/Likepage.js:5]: ok will do once every issues are resolved.

**Pull request merged**